### ISSUE TITLE
Evaluation of the performance result 

### ISSUE THREAD

**Issue by dikubab** [whai362/pan_pp.pytorch#41]
Hello Author, 
First of all, I would like to appreciate your work and effort. I have tried your repo. The evaluation code gives me an error of the  "_The sample 199 not present in GT_," but the label text is there. When I tried to see the result via visualizing it on the images, it seems good. Let me know if there is any solution from your side.

**Comment by czczup** [comment]
Hi, @dikubab 
Can you provide more information? For example, which dataset and test script did you use.

**Comment by dikubab** [comment]
> Hi, @dikubab
> Can you provide more information? For example, which dataset and test script did you use.

It is solved. Thanks for your response.

**Comment by DenghuiXiao** [comment]
> > Hi, @dikubab
> > Can you provide more information? For example, which dataset and test script did you use.
> 
> It is solved. Thanks for your response.

How to solved I meet same problem

**Comment by zeng-cy** [comment]
> How to solved I meet same problem
How to solved I meet same problem

**Comment by zeng-cy** [comment]
> Hello Author, First of all, I would like to appreciate your work and effort. I have tried your repo. The evaluation code gives me an error of the "_The sample 199 not present in GT_," but the label text is there. When I tried to see the result via visualizing it on the images, it seems good. Let me know if there is any solution from your side.

How to solved I meet same problem

**Comment by zeng-cy** [comment]
> > Hi, @dikubab
> > Can you provide more information? For example, which dataset and test script did you use.
> 
> It is solved. Thanks for your response.
How to solved I meet same problem

**Comment by zeng-cy** [comment]
> > > Hi, @dikubab
> > > Can you provide more information? For example, which dataset and test script did you use.
> > 
> > 
> > It is solved. Thanks for your response.
> 
> How to solved I meet same problem

请问您解决了这个问题吗

**Comment by dikubab** [comment]
@zeng-cy @DenghuiXiao @czczup 
I used my dataset to train and test. The main problem was I zipped the test set in a regular way. The problem was solved zipping with a command like this “zip -9jpr submit.zip submit/*txt “

**Comment by zeng-cy** [comment]
Hello, I really appreciate your answer. I also used my own dataset to generate this question. However, I have a question about whether the test set you mentioned should be placed under the eval/ic folder of the project.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月25日(星期五) 晚上9:03
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [whai362/pan_pp.pytorch] Evaluation of the performance result  (#41)






@***@***.***@czczup
我用我的数据集来训练和测试。主要的问题是我用常规的方式压缩了测试集。使用如下命令“zip -9jpr submit.zip submit/*txt”来压缩这个问题就解决了。

—
直接回复此邮件，在GitHub上查看，或取消订阅.
***@***.***与&gt;.

**Comment by zeng-cy** [comment]
> 

Thank you very much. The problem has been solved. Thank you！
